src/Icon.tsx

```diff
@@ -23,9 +23,8 @@ export { IFaIconProps, FaIcon, FaIconLibrary };
  * BLUEPRINT ICON
  */
 
-interface IIcon extends BPIconProps {}
-const Icon: React.FunctionComponent<IFaIconProps> = props => {
+const Icon: React.FunctionComponent<IIconProps> = (props: IIconProps) => {
   return <BPIcon {...props} />;
 };
 
-export { Icon, IIcon, IconNames };
+export { Icon, IIconProps, IconNames };
```

Comment:
The Blueprint Icon only accepts their `IIconProps`.  Even though our sub-interface made no changes to the interface, it was still not recognized as a valid interface by the Blueprint button, so I replaced our sub-interface with Blueprint's `IIconProps`.